dialog/form.go

```diff
@@ -0,0 +1,98 @@
+package dialog
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/container"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// FormDialog is a simple dialog window for displaying FormItems inside a form.
+type FormDialog struct {
```

Comment:
Exporting this new type provides no functionality - did you consider making it a private type and having the constructors return `fyne.Dialog` instead?

Reply:
Yeah that makes more sense. :)

Reply:
Hmm, it looks like this change makes golint complain.